capstone/src/main/webapp/style.css

```diff
@@ -2,10 +2,12 @@
   border: solid black;
   border-radius: 25px;
   height: 600px;
+  margin-left: auto;
+  margin-right: auto;
   width: 600px;
 }
 
-#main-content {
+.main-content {
```

Comment:
Hey, is there a reason you changed this to be a class selector instead of an id selector? If so, can we also reflect this change in the HTML file? Or does it cause unforeseen issues?

Reply:
I changed it because when I was adding the header bar to the details and restaurant signup pages I wanted to have main-content be the id around all the content but not have the style. The main-content HTML element in index.html also has class="main-content" so I thought it'd be okay, but is there another issue with changing it that I missed?

Reply:
Oh, okay, I see. That makes sense.